customTests/azure_gpu_xid.nhc

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# reference this link for info on XID error codes: https://docs.nvidia.com/deploy/xid-errors/index.html
+
+# Check for the following GPU Xid errors in dmesg
+XID_EC="48 56 57 58 62 63 64 65 68 69 73 74 79 80 81 92 119 120"
+GPU_XID_TEST="GPU Xid errors detected"
+
+
+function check_gpu_xid()
+{
+/bin/dmesg | grep -q "Xid"
```

Comment:
Is this script run as root? In some of the newer azhpc-images dmesg requires root privileges

Reply:
Yea it is.